config/bb_config.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+import "os"
+
+type BBConfig struct {
+	URLBBToken          string
+	URLBBRegisterBoleto string
+}
+
+func getBBConfig() BBConfig {
```

Comment:
Opa, sem voltar o ponteiro, tu vai ter criado um objeto(struct), e voltar uma cópia dele 😀

```suggestion
func getBBConfig() *BBConfig {
```